### PR TITLE
Profile settings

### PR DESCRIPTION
**Pull request by xx-TOOL-xx**
Profile edit form connected to backend, added posibility to delete all education or exerience. Now all skills and categories not hardcoded, but queried from database

### CODE REVIEW

File: src/components/educationLayout/EducationLayout.tsx [hunk 1]
```diff
@@ -11,45 +10,51 @@ function EducationLayout({
...
+      jobTitle: "",
+      employer: "",
+      startDate: null,
+      endDate: null,
+      city: "",
+      description: "",
+      degree: "",
+      school: "",
+      active: true,
+    });
```
Comment (lyudad) [inline src/components/educationLayout/EducationLayout.tsx]: can we set null insted this full object?

File: src/components/profileEditForm/ProfileEditForm.tsx [hunk 2]
```diff
@@ -1,52 +1,124 @@
...
+      city: "",
+      description: "",
+      school: "",
+      endDate: new Date().toISOString(),
+      degree: "",
+      startDate: new Date().toISOString(),
+      active: false,
+      employer: "",
+      jobTitle: "",
+    }));
```
Comment (lyudad) [inline src/components/profileEditForm/ProfileEditForm.tsx:71]: dont realy like this empty objects, dos it possible to do it other way?

File: src/components/educationLayout/EducationLayout.tsx (hunk 1 above)
Comment (xx-TOOL-xx) [reply src/components/educationLayout/EducationLayout.tsx]: fixed

File: src/components/profileEditForm/ProfileEditForm.tsx (hunk 2 above)
Comment (xx-TOOL-xx) [reply src/components/profileEditForm/ProfileEditForm.tsx:71]: fixed

**Pull request merged**